docs/preview/features/secret-store/index.md

```diff
@@ -78,6 +77,34 @@ public class HealthController : ControllerBase
 }
 ```
 
+### Using secret store outside .NET hosting
+The secret store is also available directly on the `IServiceCollection` for applications that run outside the .NET hosting but still want to use the Arcus secret store.
+
+Just like you would register the secret store on the `HostBuilder`, you can use the `.AddSecretStore` extension to register the secret store:
+
+```csharp
+IServiceCollection services = ...
+IConfiguration configuration = ...
+
+// Mark: the secret store uses logging, so make sure that you have set the logging beforehand.
+service.AddLogging(...);
+
+services.AddSecretStore(stores =>
+{
+    stores.AddEnvironmentVariables();
+    
+    #if DEBUG
+    builder.AddConfiguration(configuration);
+    #endif
+
+    var keyVaultName = configuration["KeyVault_Name"];
+    stores.AddAzureKeyVaultWithManagedServiceIdentity($"https://{keyVaultName}.vault.azure.net");
+});
+```
+
+When the dependency injection container injects the dependent services in the rest of your application, 
```

Comment:
This is a bit too much implementation detail imo, what about this:
```suggestion
When your application wants to access a secret, all it has to do is use `ISecretProvider` which will give you access to all the registered secret providers.
```

What do you think?

Reply:
Yes, no problem!